packages/validator/src/services/prepareBeaconProposer.ts

```diff
@@ -1,9 +1,10 @@
 import {Epoch, bellatrix} from "@lodestar/types";
-import {Api, ApiError, routes} from "@lodestar/api";
-import {BeaconConfig} from "@lodestar/config";
+import {Api, ApiError} from "@lodestar/api";
 import {SLOTS_PER_EPOCH} from "@lodestar/params";
+import {BeaconConfig} from "@lodestar/config";
 
-import {IClock, LoggerVc, batchItems} from "../util/index.js";
+import {ProposerPreparationData} from "@lodestar/api/lib/beacon/routes/validator.js";
```

Comment:
same as above, import from `"@lodestar/api"` and should avoid deep imports from  `lib` folder of packages